tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "esnext",
+    "target": "es2019",
```

Comment:
Check that the coverage line numbers are correct with this setting (otherwise lowering the target might work). I seem to remember stumbling upon problems people have had in this regard.

Reply:
Did a quick test, and they seem to work as expected.